src/Domain/FunctionsOfTime/Tags.cpp

```diff
@@ -31,23 +36,29 @@ auto InitialFunctionsOfTime<Dim, true>::create_from_options(
     std::unordered_map<std::string,
                        domain::FunctionsOfTime::PiecewisePolynomial<max_deriv>>
         spec_functions_of_time{};
-    importers::read_spec_third_order_piecewise_polynomial(
+    domain::FunctionsOfTime::read_spec_third_order_piecewise_polynomial(
         make_not_null(&spec_functions_of_time), *function_of_time_file,
         *function_of_time_name_map);
 
-    std::unordered_map<std::string,
-                       std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>
-        functions_of_time{domain_creator->functions_of_time()};
-    for (auto spec_and_spectre_names : *function_of_time_name_map) {
-      const std::string& spectre_name = std::get<1>(spec_and_spectre_names);
+    auto functions_of_time{domain_creator->functions_of_time()};
```

Comment:
why not just `=`? :D

Reply:
We discussed on the call that either are preferable, and I've gotten to like brace initialization, but I don't feel strongly about it, so here I'll just change it to =

Reply:
Because it'sa constructor call and so `{...}` is actually more correct... :shrug: I'm with Geoffrey here.

Reply:
@nilsdeppe do you want me to change it back to {}, or can we let it go using = this time?

Reply:
I don't care, I just don't think we should argue for `=` syntax when it's in a sense "less correct" :shrug: In mathematical expressions I can see it being easier to read

Reply:
Since I have other changes to make anyway, I'm changing this back to {} initialization